website/content/en/docs/cli/operator-sdk_create_api.md

```diff
@@ -47,7 +50,6 @@ operator-sdk create api [flags]
 
 ```
       --controller           if set, generate the controller without prompting the user (default true)
-      --crd-version string   version of CustomResourceDefinition to scaffold. Options: [v1, v1beta1] (default "v1")
```

Comment:
The flag was marked as deprecated and because of this the generate doc is removing when ihmo should not